app/javascript/pages/Dashboard/index.js

```diff
@@ -141,12 +130,12 @@ const LeaderboardContainer = ({
         <div className={s.topVolunteers}>Top Volunteers</div>
         <Filter
           collection={offices}
-          value={dashboardOfficeFilter.value}
+          value={dashboardOfficeFilter.value === 'current' ? currentUser.office.id : dashboardOfficeFilter.value}
           onChange={(_event, _index, value) => changeDashboardOfficeFilter(value)}
           itemValueProp="name"
         />
       </div>
-      {topUsers(dashboardOfficeFilter, users).map((user, i) => (
+      {users.map((user, i) => (
         <div className={s.leaderboardUser} key={`user-${i}`}>
```

Comment:
slightly out of scope of this PR but we should just use `user.id` here, no? no need for the `index` argument then